features/machine/machine.feature

```diff
@@ -487,30 +487,24 @@ Feature: Machine features testing
   @admin
   @4.10 @4.9
   Scenario: Node labels and Affinity definition in PV should match
-    Given I have a project
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
```

Comment:
Creating pvc/pods should not require admin privilege.

Reply:
acked